lib/cli/ui/prompt/interactive_options.rb

```diff
@@ -52,7 +52,13 @@ def initialize(options, multiple: false)
           @filter = ''
           # 0-indexed array representing if selected
           # @options[0] is selected if @chosen[0]
-          @chosen = Array.new(@options.size) { false } if multiple
+          if multiple
+            @chosen = if default
+              @options.map { |option| default.include?(option) }
+            else
+              Array.new(@options.size) { false }
+            end
+          end
```

Comment:
If the default value of `default` is `[]` instead of `nil`, it's more accurate, and allows
```suggestion
          @chosen = @options.map { |option| default.include?(option) } if multiple
```

Reply:
That would be cleaner indeed! The problem with that is that I'd need to change the default value up the stack as well (e.g. in `Prompt.ask`), which makes less sense:

```ruby
def internal(default: [])
  default.class
end

def external(default: nil)
  internal(default: default)
end

internal # Array
external # NilClass
```